lib/caasp.pm

```diff
@@ -75,12 +75,11 @@ sub process_reboot {
     power_action('reboot', observe => !$trigger, keepconsole => 1);
 
     # No grub bootloader on xen-pv
-    unless (check_var('VIRSH_VMM_TYPE', 'linux')) {
-        assert_screen 'grub2', 90;
-        send_key 'ret';
-    }
+    # No grub2 needle because of a stall - poo#28648
```

Comment:
Please, remove commented code, I guess we also could add soft-failure as it's workaround.

Reply:
I removed commented code, but I would like to keep note about why not to match grub2 needle there.